hana/lib/HANAService.js

```diff
@@ -60,7 +60,12 @@ class HANAService extends SQLService {
           HANAVERSION = dbc.server.major
           return dbc
         } catch (err) {
-          if (!isMultitenant || err.code !== 10) throw err
+          if (isMultitenant) {
```

Comment:
Should we drain the pool in addition as otherwise already opened connections would remain non-functional?

Reply:
Opened connections keep working for HANA